fr/README.md

```diff
@@ -0,0 +1,115 @@
+# Licences Creative Commons pour GitHub Projects
+
+Dans ce dépot vous trouverez des manières simples pour appliquer les licences Creatives Commons sur vos propres dépots Github en langage Markdown.
+
+> **ATTIENTION :**
+> Vous ne devez pas utiliser de licence Creative Commons licenses pour des logiciels.
+> Pour cela, préférez les licences Free Software telles que GPL, BSD, etc.
+> Les licences Creative Commons s’appliquent uniquement à la propriété intellectuelle.
+
+L’approche la plus simple pour trouver la bonne licence CC pour votre projet est le [site Creative Commons](https://creativecommons.org/choose/).
+
+Nous proposons ici trois de ces licences :
+
+* [Creative Commons Attribution 4.0 International](#cc-attribution-40-international)
+* [Creative Commons Attribution-ShareAlike 4.0 International](#cc-attribution-sharealike-40-international)
+* [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International](#cc-attribution-noncommercial-sharealike-40-international)
+
+Plus d’informations à propos de ces licences ainsi qu’une en version texte brute disponibles sur https://choosealicense.com/.
+
+La version vectiorielle des images Creative Commons et autres sont disponibles sur https://creativecommons.org/about/downloads/.
+
+## CC Attribution 4.0 International
+
+Pour utiliser la licence CC-BY, ajoutez le texte suivant à votre fichier`README.md` ainsi que le [fichier texte](LICENSE-CC-BY) corresponant renommé `LICENSE`.
+
+```markdown
+Shield : [![CC BY 4.0][cc-by-shield]][cc-by]
+
+Cette œuvre est mise à disposition selon les termes de la
+[Creative Commons Attribution 4.0 International License][cc-by].
+
+[![CC BY 4.0][cc-by-image]][cc-by]
+
+[cc-by]: http://creativecommons.org/licenses/by/4.0/
```

Comment:
The links to the licenses descriptions should point to their translated versions. For example, this link should point to https://creativecommons.org/licenses/by/4.0/deed.fr

Reply:
Done!